### PR TITLE
Updates to include deploying sqlite staging contract

### PR DESCRIPTION
**Pull request by joewagner**
This also includes a fix to the verify script

### CODE REVIEW

File: scripts/verify.ts [hunk 1]
```diff
@@ -20,7 +20,6 @@ async function main() {
   const impl = await upgrades.erc1967.getImplementationAddress(tables.address);
   await run("verify:verify", {
     address: impl,
-    constructorArguments: [baseURI],
```
Comment (joewagner) [inline scripts/verify.ts:23]: This was a copy->paste typo, there shouldn't be a constructor argument

File: hardhat.config.ts [hunk 2]
```diff
@@ -104,6 +104,15 @@ const config: HardhatUserConfig = {
           ? [process.env.OPTIMISM_KOVAN_STAGING_PRIVATE_KEY]
           : [],
     },
+    "optimism-kovan-staging-sqlite": {
+      url: `https://opt-kovan.g.alchemy.com/v2/${
+        process.env.OPTIMISM_KOVAN_STAGING_API_KEY ?? ""
+      }`,
+      accounts:
+        process.env.OPTIMISM_KOVAN_STAGING_PRIVATE_KEY !== undefined
```
[the remark this answers is not in the document]
Comment (joewagner) [reply hardhat.config.ts:112]: either way it probably doesn't hurt to use a different env var.  I'll update this.

File: scripts/deploy.ts [hunk 3]
```diff
@@ -37,7 +37,7 @@ async function main() {
 
   // Create health bot table
   const { chainId } = await account.provider.getNetwork();
-  const createStatement = `create table healthbot_${chainId} (counter bigint);`;
+  const createStatement = `create table healthbot_${chainId} (counter integer);`;
```
Comment (joewagner) [inline scripts/deploy.ts:40]: @jsign double checking this is correct?

Comment (jsign) [reply scripts/deploy.ts:40]: Yep

**Review by sanderpick — APPROVED** [review]
LGTM! Sounds like we'll replace staging w/ the SQLite version soon, but we can swap back then.

**Pull request merged**